native_app/README.md

```diff
@@ -0,0 +1,40 @@
+# Native compilation sample
+
+This sample is a command line application that can be compiled to native code
+using dart2native. The application fetches stats for a GitHub user and prints
+them to the console.
+
+This sample also shows how to parse command line options into Dart object using
+[`package:build_cli_annotations`][build-cli] and [`package:build`][build].
+
+## Building
+Use the `dart2native` command included in Dart 2.6:
+
+### Linux and macOS
+
+```
+dart2native bin/github_activity.dart -o github_activity
+```
+
+## Windows
+
+```
+dart2native bin/github_activity.dart -o github_activity.exe
```

Comment:
Is the `-o` necessary? (Probably a good idea, but I seem to remember that the default name has an `.exe` suffix.)

Reply:
Ah, but I think it'd go into the `bin` directory, which apparently is not what you want.

Reply:
My thinking was to demonstrate that the flag exists, but I'll add a line to explain

Reply:
I remember now why I wrote it this way. By default, dart2native appends an `.exe` to the end of the file, but I don't think this is the standard convention in Unix. @mit-mit what do you think?

Reply:
I don't think there are any standards on Linux. For comparison, kotlinc uses .kexe on linux and .exe on windows.

I'd just leave it at .exe as it (in my personal preference) helps explain what the file is.

Reply:
Please do *not* generate executable files with `.exe` extension on linux and macOS. Nothing screams DOS like `.exe`. The standard binary extension on macOS and linux is nothing, because we have `chmod +x`.

Reply:
I've updated the README to use an empty extension on Linux and macOS for now.